examples/docker/plasticc-on-omnisci/plasticc-omnisci.py

```diff
@@ -118,7 +119,7 @@ def multi_weighted_logloss(y_true, y_preds, classes, class_weights):
     multi logloss for PLAsTiCC challenge
     """
     y_p = y_preds.reshape(y_true.shape[0], len(classes), order="F")
-    y_ohe = pd.get_dummies(y_true)
+    y_ohe = pandas.get_dummies(y_true)
```

Comment:
why?

Reply:
This operation is performed very quickly on Pandas (something around 0.001sec). When using Modin, the time increases due to overhead costs.

Reply:
I suggest that you create an issue for this because Modin is supposed to be faster and using pandas in an example for a reason that it is faster produces a bad impression.

Reply:
The change was reverted.

Found bug and reported - #3368.